app/facades/food_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodFacade
+  require "./app/services/food_service.rb"
+
+  def self.get_food_items(food)
+    items = FoodService.call_for_items(food)
+    items[:foods][0..9].map do |data|
```

Comment:
Glad to see you only ever creating 10 FoodItem objects. This is efficient and how it should be done. I would break up the `map` and the `items` call to the service into two methods, so that I could thoroughly test both.